pom.xml

```diff
@@ -93,16 +93,16 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>2.3.2</version>
+                <version>3.11.0</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>11</source>
```

Comment:
Source and target should both be set back to 1.8.

The changes from #761 will also need to be incorporated once merged to master